subsys/bluetooth/mesh/cfg_cli.c

```diff
@@ -1125,6 +1146,44 @@ int bt_mesh_cfg_app_key_add(uint16_t net_idx, uint16_t addr, uint16_t key_net_id
 	return cli_wait();
 }
 
+int bt_mesh_cfg_node_reset(uint16_t net_idx, uint16_t addr, bool *status)
+{
+	BT_MESH_MODEL_BUF_DEFINE(msg, OP_NODE_RESET, 2);
```

Comment:
Config Node Reset no need for payloads, of course, the length.
`BT_MESH_MODEL_BUF_DEFINE(msg, OP_NODE_RESET, 0);`